package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "calibre",
-  "version": "2.4.0",
+  "version": "3.0.0",
   "engines": {
     "node": "> 8.15.0"
```

Comment:
Going to bump to a recent LTS node?

Reply:
10 will be supported through 2021-04-30. Should we go with that or are you thinking even more recent?

Reply:
10+ 👍